source/hyperbolic_module.template.h

```diff
@@ -627,7 +627,9 @@ namespace ryujin
 #endif
 
             const auto c_ij = cij_matrix.template get_tensor<T>(i, col_idx);
-            const auto scaled_c_ij = c_ij / d_ij;
+            const auto regularization =
+                T(100. * std::numeric_limits<Number>::min());
```

Comment:
Just curious, why use `T()` instead of `Number()`?

Reply:
Because the local type which we have to construct in the `loop` statement is `T` and not `Number` (which is either `double` or `float`). Admittedly, this is a bit confusing.

Reply:
Thank you for clarifying!